client/packages/inventory/src/Stocktake/DetailView/ContentArea/columns.ts

```diff
@@ -49,8 +50,10 @@ export const useStocktakeColumns = ({
   onChangeSortBy,
 }: UseStocktakeColumnOptions): Column<
   StocktakeLineFragment | StocktakeSummaryItem
->[] =>
-  useColumns<StocktakeLineFragment | StocktakeSummaryItem>(
+>[] => {
+  const { getError } = useStocktakeLineErrorContext();
```

Comment:
I didn't want to use context at first, but realised it's much more convenient

Reply:
See comment in context about the reasoning behind context vs extra field on stocktake line or draft stocktake line